packages/vscode-graphql-syntax/jest.config.js

```diff
@@ -0,0 +1 @@
+module.exports = require('../../jest.config.base')(__dirname, 'node');
```

Comment:
can you run these tests with vitest? you need to make the same as I made in another PR https://github.com/graphql/graphiql/pull/3285

1. modify root `package.json`

`"test:ci": "yarn build && jest --coverage && yarn workspace vscode-graphql-syntax test"`

https://github.com/B2o5T/graphiql/blob/ee52a1be7d9f99abf8884907a122126f8777d261/package.json#L80

2. modify this action
<img width="343" alt="image" src="https://github.com/graphql/graphiql/assets/7361780/542fc838-d1e4-4351-99bb-27e13c94488c">

3. and add test command
<img width="188" alt="image" src="https://github.com/graphql/graphiql/assets/7361780/bb126cce-b898-4835-a630-e838439328bb">

Reply:
done